connecting-to-testnet/building-binary.md

```diff
@@ -1,66 +1,67 @@
 
-```
-git clone --branch v1.0.1 https://github.com/b-harvest/gravity-dex
-```
+- `cosmos/gaia` the source code behind Cosmos Hub
+- `tendermint/liquidity` the module source code of Gravity DEX 
 
-We're using `b-harvest/gravity-dex`, which is `cosmos/gaia` (the source code behind Cosmos Hub) with the `tendermint/liquidity` module (source code of Gravity DEX) imported.
+1. Clone the code repository:
 
-Install node's binary `gaiad`:
+    ```
+    git clone --branch v1.0.1 https://github.com/b-harvest/gravity-dex
+    ```
 
-```
-cd gravity-dex && make install
-```
+1. Install the node's binary `gaiad`:
+
+    ```
+    cd gravity-dex && make install
+    ```
 
 ## Initializing a Node
 
-Initialize your node (replace `YOUR_MONIKER` with any string that identifies your node):
+To initialize your node:
 
 ```
 gaiad init YOUR_MONIKER --chain-id cosmoshub-testnet
 ```
+ 
+where `YOUR_MONIKER` is a placeholder for a string that identifies your node. Be sure to replace `YOUR_MONIKER` with a self-describing string for your node. 
 
 ## Starting a Node
 
-Download the genesis file and save it into the data directory of your node:
+1. Download the genesis file:
 
-```
-curl -s https://rpc.testnet.cosmos.network/genesis | jq -r .result.genesis > ~/.gaia/config/genesis.json
-```
+    ```
+    curl -s https://rpc.testnet.cosmos.network/genesis | jq -r .result.genesis > ~/.gaia/config/genesis.json
+    ```
+1. Save the genesis file in the data directory of your node.
 
-Add seed nodes:
+1. Add seed nodes using the command that is suitable for your environment. 
 
-```
-sed -i 's/seeds =.*/seeds = "30dce239c56cfd25cf0cebf5726c7c8be1e46f66@104.196.38.186:31819,754a5f864adc5a60b287d4aed4f0ab11d8b056c8@34.73.24.113:31654"/g' ~/.gaia/config/config.toml
-```
+    - For operating systems other than macOS or BSD:
 
-If you're using macOS or BSD, use the following command:
+    ```
+    sed -i 's/seeds =.*/seeds = "30dce239c56cfd25cf0cebf5726c7c8be1e46f66@104.196.38.186:31819,754a5f864adc5a60b287d4aed4f0ab11d8b056c8@34.73.24.113:31654"/g' ~/.gaia/config/config.toml
+    ```
 
-```
-sed -i '' 's/seeds =.*/seeds = "30dce239c56cfd25cf0cebf5726c7c8be1e46f66@104.196.38.186:31819,754a5f864adc5a60b287d4aed4f0ab11d8b056c8@34.73.24.113:31654"/g' ~/.gaia/config/config.toml
-```
+    - For macOS or BSD operating systems:
 
-Start the node:
+    ```
+    sed -i '' 's/seeds =.*/seeds = "30dce239c56cfd25cf0cebf5726c7c8be1e46f66@104.196.38.186:31819,754a5f864adc5a60b287d4aed4f0ab11d8b056c8@34.73.24.113:31654"/g' ~/.gaia/config/config.toml
+    ```
 
-```
-gaiad start
-```
+1. Start the node:
+
+    ```
+    gaiad start
+    ```
 
-Your node should start syncing and over time will catch up with the network. By default, your node's RPC is accessible on [http://localhost:26657](http://localhost:26657).
+Your node starts syncing and over time catches up with the network. By default, your node's RPC is accessible on [http://localhost:26657](http://localhost:26657).
```

Comment:
I couldn't figure out how to tell when my node was "caught up"?

Reply:
Did my edit for this not make it?

Check https://api.testnet.cosmos.network/blocks/latest and find the data `height`. Compare this with your local chain, you can get the information with `gaiad status` and look for the data `latest_block_height`.

Reply:
I wonder if instead users can open `http://localhost:26657/status` and look for `result.sync_info.catching_up`? Is 26657 working while the node is catching up?

Reply:
<img width="1304" alt="Screenshot 2021-05-18 at 14 59 33" src="https://user-images.githubusercontent.com/332151/118632383-cbd61500-b7e9-11eb-8b3b-0a84c55854b8.png">